src/templates/amfcfg.conf.j2

```diff
@@ -1,11 +1,11 @@
 configuration:
-  amfDBName: {{ amf_database_name }}
+  amfDBName: {{ database_name }}
   amfName: AMF
   debugProfilePort: 5001
   enableDBStore: false
   enableSctpLb: false
   mongodb:
-    name: {{ default_database_name }}
+    name: free5gc
```

Comment:
is this needed?

Reply:
Good catch, tested without and it seems like it's working